rtmidi17/detail/winmm.hpp

```diff
@@ -98,6 +95,10 @@ class observer_winmm final : public observer_api
     inline static const bool OUTPUT = false;
 
 public:
+    using PortList = std::vector<std::string>;
+
+    PortList inputPortList;
```

Comment:
the typedef can stay public, but I think that the member variables should stay private